ocs/defaults.py

```diff
@@ -14,22 +14,15 @@
 import os
 from getpass import getuser
 
+from ocs.constants import TEMPLATES_DIR
 from utility.templating import load_yaml_to_dict
 
-THIS_DIR = os.path.dirname(os.path.abspath(__file__))
-TOP_DIR = os.path.dirname(THIS_DIR)
-TEMPLATE_DIR = os.path.join(TOP_DIR, "templates/ocs-deployment/")
+DEPLOYMENT_TEMPLATE_DIR = os.path.join(TEMPLATES_DIR, "ocs-deployment")
```

Comment:
Could you add below 2 so that most of test cases benifits:

TEMPLATE_DIR_RBD = os.path.join(TOP_DIR, "templates/CSI/rbd/")
TEMPLATE_DIR_CEPHFS = os.path.join(TOP_DIR, "templates/CSI/cephfs/")

Reply:
Yeah I see that vasu has added a number of new constants in for those directories, if we decide to move forward with the constants module I will add those all in.